src/pages/ProfilesExplorerView/components/SceneProfilesExplorer/SceneProfilesExplorer.tsx

```diff
@@ -361,154 +355,30 @@ export class SceneProfilesExplorer extends SceneObjectBase<SceneProfilesExplorer
-    const styles = useStyles2(getStyles, chromeHeaderHeight ?? 0); // eslint-disable-line react-hooks/rules-of-hooks
-    const { data, actions } = model.useProfilesExplorer();
-
-    const {
-      explorationType,
-      dataSourceVariable,
-      timePickerControl,
-      refreshPickerControl,
-      sceneVariables,
-      gridControls,
-      body,
-      dataSourceUid,
-    } = data;
+    const styles = useStyles2(getStyles); // eslint-disable-line react-hooks/rules-of-hooks
+
+    const { data } = model.useProfilesExplorer();
+    const { body, dataSourceUid } = data;
 
     return (
       <GitHubContextProvider dataSourceUid={dataSourceUid}>
-        <div className={styles.header} data-testid="allControls">
-          <GiveFeedbackButton />
-
-          <div className={styles.controls} data-testid="appControls">
-            <div className={styles.headerLeft}>
-              <div className={styles.dataSourceVariable}>
-                <InlineLabel width="auto">{dataSourceVariable.state.label}</InlineLabel>
-                <dataSourceVariable.Component model={dataSourceVariable} />
-              </div>
-
-              <ExplorationTypeSelector
-                options={SceneProfilesExplorer.EXPLORATION_TYPE_OPTIONS}
-                value={explorationType as string}
-                onChange={actions.onChangeExplorationType}
-              />
-            </div>
-
-            <div className={styles.headerRight}>
-              {timePickerControl && (
-                <timePickerControl.Component key={timePickerControl.state.key} model={timePickerControl} />
-              )}
-              {refreshPickerControl && (
-                <refreshPickerControl.Component key={refreshPickerControl.state.key} model={refreshPickerControl} />
-              )}
-
-              <div className={styles.miscButtons}>
-                <IconButton name="cog" tooltip="View/edit user settings" onClick={actions.onClickUserSettings} />
-
-                <IconButton
-                  name="share-alt"
-                  tooltip="Copy shareable link to the clipboard"
-                  onClick={actions.onClickShareLink}
-                />
-
-                <PluginInfo />
-              </div>
-            </div>
-          </div>
-
-          <div id={`scene-controls-${explorationType}`} className={styles.sceneControls} data-testid="sceneControls">
-            {sceneVariables.map((variable) => (
-              <div key={variable.state.name} className={styles.variable} data-testid={variable.state.name}>
-                {variable.state.label && <InlineLabel width="auto">{variable.state.label}</InlineLabel>}
-                <variable.Component model={variable} />
-              </div>
-            ))}
-
-            {gridControls.map((control) => (
-              <control.Component key={control.state.key} model={control} />
-            ))}
-          </div>
-        </div>
+        <Header model={model} />
```

Comment:
It feels pretty heavy to pass the whole model for now. In the future, we might want to consider converting `Header` to a Scene object.